### PR TITLE
added missing challenge 

### PR DESCRIPTION
**Pull request by devkennyy**
also added messages for accuracy tests and autorole for accuracy test

### CODE REVIEW

File: frontend/static/challenges/_list.json [hunk 1]
```diff
@@ -358,6 +370,13 @@
     "type": "script",
     "parameters": ["shrek.txt",null,null]
   }
+  ,{
```
Comment (Bruception) [inline frontend/static/challenges/_list.json:373]: What is this?

Comment (devkennyy) [reply frontend/static/challenges/_list.json:373]: The same thing you see on line 175 and line 191

Comment (Bruception) [reply frontend/static/challenges/_list.json:373]: I mean the challenge? Was adding this new challenge discussed? Also the related file: `sourcecode.txt` is empty

Comment (devkennyy) [reply frontend/static/challenges/_list.json:373]: This challenge was already made, it was just missing from the site. Fixing the empty file now!

**Review by Miodec — APPROVED** [review]
Thanks

**Pull request merged**